config/build/tailwind.js

```diff
@@ -25,14 +32,57 @@ module.exports = {
 		},
 		iconSizes: {
 		},
+		fontFamily: {
+            sans: [
+                'Arial',
+                'sans-serif'
+            ],
+        },
+		fontSize: {
+            'tiny': convertToRem(12),
+            'xs':   convertToRem(13),
+            'sm':   convertToRem(14),
+            'base': convertToRem(16), 
+            'md':   convertToRem(18), 
+            'lg':   convertToRem(20), 
+            'xl':   convertToRem(30),  
+            '2xl':  convertToRem(35),
+            '3xl':  convertToRem(40), 
+        },
+		fontWeight: {
+            light:      300,
+            normal:     400,
+            medium:     500,
+            semibold:   600,
+            bold:       700,
+        },
+        lineHeight: {
+            'none': '1',
+            '1':    '1.2',
+            '2':    '1.4',
+            '3':    '1.6',
+            '4':    '1.75',
+            '5':    '2',
+        },
```

Comment:
Let's update the keys here to the percentage equivalent of the values:
```
lineHeight: {
    'none': '1',
    '120':  '1.2',
    '140':  '1.4',
},
```
We'll want these to be setup for every 1/10th from 1 to 2.5...